.travis.yml

```diff
@@ -10,7 +10,7 @@ matrix:
       php: 5.3
     - env: DM=~4.2
       php: 5.4
-    - env: DM=~4.2
+    - env: DM=~4.3@dev
```

Comment:
we already test against DM@dev below with the same configuration. The dev version should always allow failures as it shouldn't block pull requests on this component.